kits/arm/ex_mobile_device_control_v2.py

```diff
@@ -0,0 +1,294 @@
+#!/usr/bin/env python3
+
+import hebi
+import numpy as np
+import os
+import sys
+
+
+# ------------------------------------------------------------------------------
+# Add the root folder of the repository to the search path for modules
+root_path = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..'))
+sys.path = [root_path] + sys.path
+# ------------------------------------------------------------------------------
```

Comment:
This is the hack necessary to add in the `util` package for stuff like the `quat2rot` functions.